### PR TITLE
Updates rack dependency in .gemspec

### PR DESCRIPTION
**Pull request by towanda**
See https://github.com/ruby-grape/grape/issues/1066

### CODE REVIEW

**Comment by dblock** [comment]
Build needs fixing, please.

File: CHANGELOG.md [hunk 1]
```diff
@@ -20,7 +20,7 @@ Next Release
 * [#1052](https://github.com/ruby-grape/grape/pull/1052): Reset `description[:params]` when resetting validations - [@marshall-lee](https://github.com/marshall-lee).
 * [#1088](https://github.com/ruby-grape/grape/pull/1088): Support ActiveSupport 3.x by explicitly requiring Hash#except - [@wagenet](https://github.com/wagenet).
 * [#1096](https://github.com/ruby-grape/grape/pull/1096): Fix coercion on booleans - [@towanda](https://github.com/towanda).
-
```
Comment (dblock) [inline CHANGELOG.md]: I liked my line! Also amend with the actual issue and lets say something like "Rack X.Y.Z is now required" instead.

**Comment by towanda** [comment]
This was fixed in http://en.battleship-game.org/id17445101, thanks @dmitry :)

**Comment by dblock** [comment]
@towanda That battleship game looks RAD, but probably a bad link for Grape ;)

**Comment by towanda** [comment]
@dblock ouch! wrong copy paste ;)

**Pull request closed without merging**